spa/cypress/e2e/02-donationPageEdit.cy.js

```diff
@@ -182,7 +182,7 @@ describe('Contribution page edit', () => {
       cy.getByTestId('element-properties').should('exist');
     });
 
-    describe('Frequency editor', () => {
+    describe.only('Frequency editor', () => {
```

Comment:
Forgot to remove the `only`
```suggestion
    describe('Frequency editor', () => {
```

Reply:
Yep, faceplanted on this one. Fixed in 5c868b0c99f866bfd5758299f2286ff16a187648